composer.json

```diff
@@ -67,17 +66,20 @@
         "phpunit/phpunit": "^8.0"
     },
     "scripts": {
+        "auto-scripts": [
+            "@php app/console cache:clear",
+            "@php app/console assets:install --symlink --relative web"
```

Comment:
Since the distribution bundle isn't compatible with Symfony 4, I moved the commands it was executing out to be triggered directly.  `app/bootstrap.cache.php` isn't generated anymore either.

Reply:
why not keeping these in `symfony-scripts` though ?

Using the `auto-scripts` name to match Flex looks like a bad idea to me, because Flex also manages a custom structure for the `auto-scripts` key, while you use the standard Composer feature. So this might cause incompatibility with a Flex plugin.

Reply:
Originally I tried flat out copying an `auto-scripts` from another project not realizing the syntax for that comes from Flex, so when changed it to straight up `@php` calls I didn't think twice about moving it back.

Reply:
The scripts are moved back under `symfony-scripts`